### PR TITLE
Add SoSe 2021 Team to the credits

### PR DESCRIPTION
**Pull request by bewee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,11 +59,17 @@ Use the Test Runner Tool in your Squeak Image and select all *AlgernonTests*. No
...
+*  [Franz Sauerwald](https://github.com/FranzSw)
 *  [Jannis Rosenbaum](https://github.com/sinnaj-r)
 *  [Jonathan Schneider](https://github.com/jonaschn)
 *  [Kai Robert Kirsten](https://github.com/robertkirsten)
 *  [Kay Erik Jenß](https://github.com/kej-jay)
+*  [Laura Meister](https://github.com/dieknolle3333)
 *  [Louis Kirsch](https://github.com/timediv)
+*  [Niklas Mohrin](https://github.com/niklasmohrin)
+*  [Nils Lißner](https://github.com/TheGrayStone)
+*  [Olli Wehr](https://github.com/Olliwehr)
```
Comment (Olliwehr) [inline README.md]: Uff 😄

Comment (bewee) [reply README.md]: All intended

Comment (Olliwehr) [reply README.md]: Do you want to troll me even further? xD

File: README.md [hunk 2]
```diff
@@ -59,11 +59,17 @@ Use the Test Runner Tool in your Squeak Image and select all *AlgernonTests*. No
...
+*  [Benedikt Weber](https://github.com/bewee)
+*  [Franz Sauerwald](https://github.com/FranzSw)
 *  [Jannis Rosenbaum](https://github.com/sinnaj-r)
 *  [Jonathan Schneider](https://github.com/jonaschn)
 *  [Kai Robert Kirsten](https://github.com/robertkirsten)
 *  [Kay Erik Jenß](https://github.com/kej-jay)
+*  [Laura Meister](https://github.com/dieknolle3333)
 *  [Louis Kirsch](https://github.com/timediv)
+*  [Niklas Mohrin](https://github.com/niklasmohrin)
+*  [Nils Lißner](https://github.com/TheGrayStone)
```
Comment (TheGrayStone) [inline README.md]:
```suggestion
*  [Nils Lissner](https://github.com/TheGrayStone)
```

Comment (TheGrayStone) [reply README.md]: Please don't use the dreaded `ß` here.

**Review by TheGrayStone — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (FranzSw) [reply README.md]: @TheGrayStone really wants to change their last name

**Comment by bewee** [comment]
I really didn't expect this to be so difficult to get right^^

**Review by FranzSw — APPROVED** [review]

**Review by niklasmohrin — APPROVED** [review]

**Review by TheGrayStone — APPROVED** [review]

**Pull request merged**